### ISSUE TITLE
SILAL

### ISSUE THREAD

**Issue by johnbent** [tekinged/missing#2]
The phrase "silal mlo meringel a bedengek" shows up in two songs:  Watasi no Kot er a Soak (composed by Bandarii Olebuu) and MIK (composer unknown, but sung by Christa Mersai).  Yoichi Rengiil and Ongerung Kambes Kesolei both agreed that silal is a word that adds emphasis to the words that follow, but couldn't come up with an English equivalent.  

Is this word documented anywhere and can anyone give a better definition?

**Comment by smith-371** [comment]
hi John.

i'm completely sure it is a past form of KMAL 'very'.

**Comment by smith-371** [comment]
https://tekinged.com/misc/all_paltext.php here there are some other examples of its usage

**Comment by johnbent** [comment]
<img width="1214" alt="image" src="https://user-images.githubusercontent.com/2047294/156854643-1a21251a-ec55-4db9-b89f-d8a50e58cce9.png">

**Comment by smith-371** [comment]
Palauan S regularly changes to K before an L as in sils > klsel, meluches > lechukl. here it appears something like that.

**Comment by smith-371** [comment]
so, i guess that 'silal mlo meringel a bedengek' = my body hurt very much (in the past)

**Comment by johnbent** [comment]
Charlene was not familiar with this word but she read those sentences and agrees with this definition. Thanks @smith-371 !

**Comment by smith-371** [comment]
yep, this is not an often used word. also in its hypo form S appears insted of K: 

l=sal ~ k<m>al 

where <m> is verb marker which is deleted in hypo forms.
